src/sinks/prometheus/collector.rs

```diff
@@ -938,4 +938,31 @@ mod tests {
             "#}
         );
     }
+
+    /// According to the [spec](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md?plain=1#L115)
+    /// > Label names MUST be unique within a LabelSet.
+    /// Prometheus itself will reject the metrict with an error. Largely to remain backward compatible with older versions of Vecotr,
+    /// we only pushlish the last tag in the list.
```

Comment:
```suggestion
    /// Prometheus itself will reject the metric with an error. Largely to remain backward compatible with older versions of Vector,
    /// we only take the last tag in the list.
```

Reply:
Wow I really can't tpye today..

Reply:
Sinks publish tags, not take them - so I think we should keep publish..

Reply:
Yeah, I had it `publish` originally (assumed that's what "pushlist" was) but kept editing to make things the same